packages/ui/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+/*
+* Interval used to poll data that can be updated in the background. Ex: Document List
+* */
+export const POLLING_INTERVAL = 5000;
```

Comment:
Minor comment but I would add the unit here, either as a comment:

```suggestion
export const POLLING_INTERVAL = 5000;  // milliseconds
```

Or in the constant name:


```suggestion
export const POLLING_INTERVAL_MS = 5000;
```